image/src/app.js

```diff
@@ -1,32 +1,41 @@
 const mqtt = require('mqtt')
 const { broker, port, topic, protocol } = require('./config/broker')
 const { urls } = require('./config/nextContainer.js')
-const { module_name, module_type } = require('./config/thisContainer.js')
 const fetch = require('node-fetch')
 
-const client = mqtt.connect(broker, {
-  port,
-  protocol,
-})
+if(broker.includes('tcp') || broker.includes('tls') || broker.includes('ws') || broker.includes('wss') || broker.includes('wxs') || broker.includes('alis')) {
+  console.error("Unsupported url protocol. Please provide mqtt:// or mqtts:// urls.")
```

Comment:
idk javascript, but it would be nice to have this line terminate the app instead of having a huge `else` block